app/src/main/res/layout/item_spinner_small.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:paddingLeft="@dimen/dp_5"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingLeft` you should probably also define `paddingRight` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.